packages/components/src/components/primitives/Link/Link.test.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { screen, renderWithProviders } from 'tests/renderWithProviders';
+
+import { Link } from './Link';
+
+const icon = <i data-testid="testId" />;
+const textElement = <span data-testid="testId">span component</span>;
+const suiteSizes = ['s', 'm', 'l'];
```

Comment:
Couldn't we reuse the sizes defined in our types? `packages/components/src/components/primitives/Icons/types.ts`